doc/arch/overview.rst

```diff
@@ -0,0 +1,38 @@
+==========
+ Overview
+==========
+
+SEAL-Tk is designed to provide a flexible framework
+for creating GUI visualization and analysis applications
+for computer vision tools and algorithms.
+Its overall design is based on lessons learned
+developing previous generations of such tools,
+especially the ViViA_ suite of applications.
+
+Particular design goals include:
+
+  - The framework should support remove of a data source
+    from the collection of active data sources.
+
+  - The framework should not be limited to a single set
+    of associated data sources.
+
+  - The framework should support concurrent display of multiple data sets.
+
+  - The framework should support multiple associated imagery (video) sources
+    within a single data set.
+
+SEAL-Tk attempts to follow the MVF (model, view, controller) design paradigm.
```

Comment:
MVF? I believe you meant MVC.

Reply:
No, I meant "MVF", but forgot to fix the parenthetical :stuck_out_tongue:.